RFC-010-SWAP-Omni-Layer.md

```diff
@@ -0,0 +1,117 @@
+# The Omni Layer Token Asset
+
+- RFC-Number: 010
+- Status: Draft
+- Discussion-issue: -
+- Created on: 27 Mar. 2019
+
+## Table of Contents
+
+<!-- toc -->
+
+  * [Description](#description)
+  * [The Omni Layer Assets](#the-omni-layer-assets)
+    + [`quantity`](#quantity)
+    + [`property_id`](#property_id)
+  * [Registry extension](#registry-extension)
+- [Examples](#examples)
+
+<!-- tocstop -->
+
+## Description
+
+This RFC specifies how Bitcoin based Omni Layer Assets MUST be described in [RFC002](./RFC-002-SWAP.md) SWAP messages.
+OmniLayer refers specifically to [Omni Protocol](https://github.com/OmniLayer/spec).
+As required by [RFC002](./RFC-002-SWAP.md), this RFC defines the *asset* value `omni_layer`.
+This in combination with subsequent RFCs that define concrete execution steps for particular SWAP protocols enable users to negotiate and execute Omni Layer Token swaps.
+
+## The Omni Layer Assets
+
+The Omni Protocol defines several _cryptocurrencies_, also commonly referred as _coins_ or _tokens_.
+It also allow the definition of new tokens by the mean of a Bitcoin transaction containing specific data.
+
+This asset MUST be used on the Bitcoin Ledger, as defined in [RFC-004](./RFC-004-SWAP-Bitcoin.md).
+Subsequent RFCs MAY introduce the support of other ledgers for the Omni Layer token Asset.
+
+While *Omni* or *OmniCoin* holds a special place among the tokens as being predefined and having special properties in the Omni Protocol, it is not different than other Omni Layer tokens in the context of the COMIT protocol family.
+
+Each Omni Layer token defines its name.
```

Comment:
I don't think this sentence is useful for an implementor. (I know I added it in ERC20 but I don't think it helps in retrospect).